frontend/apps/vara-man/src/app/utils/index.ts

```diff
@@ -101,3 +101,7 @@ export const stringRequired: Validator = (value) => {
 	}
 	return null
 }
+
+export const prettyAddress = (address: string) => {
```

Comment:
I believe function name should start with a verb, otherwise it's hard to understand it's purpose.

`Address` in it's name seems too narrow. The scope of this function could've been text and strings in general, I don't think it'd be handy to limit it with `Address`.

`getTruncatedText`, `prettifyText` etc. should work.